tests/test_tm.py

```diff
@@ -5,15 +5,16 @@
 
 from automata.tm.dtm import DTM
 from automata.tm.ntm import NTM
+from automata.tm.mntm import MNTM
 
 
 class TestTM(object):
     """A test class for testing all Turing machines."""
 
     def setup(self):
         """Reset test machines before every test function."""
-        # DTM which matches all strings beginning with '0's, and followed by
-        # the same number of '1's
+        # DTM which matches all strings beginning with '0'N, and followed by
+        # the same number of '1'N
```

Comment:
I unintentionally did this, because I was changing some directions from `S` to `N`, since I took the example from another website, and I misused the "search and replace" function.